beacon-chain/rpc/eth/beacon/handlers.go

```diff
@@ -735,9 +755,21 @@ func (s *Server) publishBlockSSZ(ctx context.Context, w http.ResponseWriter, r *
 		httputil.HandleError(w, api.VersionHeader+" header is required", http.StatusBadRequest)
 		return
 	}
+	if !versionRequired && versionHeader == "" {
```

Comment:
Can you explain this? If a version is not required and non was given, then set a default? What happens when the version is required and they didn't provide one?